src/test/app/wasm_fixtures/fixtures.cpp

```diff
@@ -11848,25 +11848,29 @@ extern std::string const opcCallPerfTest =
     "0b0e01004180080b07623a2025660a00";
 
 extern std::string const keyletHostFunctions =
-    "0061736d01000000014d0960057f7f7f7f7f017f60047f7f7f7f017f60037f7f7f017f6008"
```

Comment:
Instead of change the whole module and re-fix all the test it is better to add small module "additional_hf_test". Then you can just play around with new functions and not messing with old.